hackathon_site/event/jinja2/event/landing.html

```diff
@@ -20,279 +20,305 @@
+                                <span>A hackathon is a big bonanza where people from everyone doing everything come together to create projects together with the newest and most exciting tech.</span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                How much does it cost?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>$0! IEEE UofT is a non-profit organization that aims to provide students accessible technical events.</span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                When can I expect to hear back after I applied?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    We are having 2 rounds of acceptance. If you applied early, you might get an email on the weekend of Oct 10th/11th. If you still haven’t gotten any replies by then,
+                                    don’t worry! We will be sending out a second round of acceptance on the weekend of Oct 24th/25th.
+                                </span>
+                            </div>
+                        </li>
+                    </ul>
                 </div>
-                <div class="scheduleRow">
-                    <p class="scheduleTime">4PM</p>
-                    <p>End</p>
+
+                <div class="col s12 m6">
+                    <ul class="collapsible">
+                        <li>
+                            <div class="collapsible-header">
+                                Who can apply? What if I have no experience?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Applicants that are currently in either highschool or university are welcomed to apply! We are a beginner friendly hackathon, so no hackathon experience is needed to
+                                    participate! We are all here to learn :)
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                What equipment and support would be provided?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Unfortunately, due to the current situation of COVID19. We won’t be able to provide any hardware. However, there would be lots of mentors ready to help you! During the
```

Comment:
This text isn't from this PR I realize, but can we change this to fix the grammar an not mention covid? Something generic is fine.